README.md

```diff
@@ -74,10 +74,12 @@ This information can be found in the Azure Portal inside your Batch Account:
 For your Azure Storage Account, we need to get:
 - Storage Account Name
 - Storage Account Access Key
+- Storage Account Endpoint Suffix
```

Comment:
nitpick: Do we _need_ this? By default we use public Azure. I think that is correct for this readme. I would have another document explaining how to use national clouds and not make the getting started more complicated.